frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -158,6 +159,11 @@ const ConceptLabel = ({
   );
 };
 
+const mark = (text: string, words: string[]): string => {
+  const regex = new RegExp(words.join("|"), "gi");
```

Comment:
Guess I would have used a `useMemo` to avoid calling `new RegExp` again and again when the `infos` change frequently on hovering around, while the `words` stay the same. But I quickly checked the performance impact and couldn't notice much. Hmm!